extension/intents/nicknames/nicknames.toml

```diff
@@ -60,4 +60,33 @@ match = """
 phrase = "delete the page called news"
 
 [[nicknames.removePageName.example]]
-phrase = "remove this tab name"
\ No newline at end of file
+phrase = "remove this tab name"
+
+[nicknames.pause]
+description = "Pauses a routine"
+match = """
+  pause routine (with |) (and |) (show |) (message | status | text | print) [message]
+  pause routine
+"""
+
+[nicknames.continue]
+description = "Continues a routine"
+match = """
+  continue routine
+"""
+
+[nicknames.startForLoop]
+description = "Statement to start loop"
+match = """
+  for each value in (clipboard) [option=clipboard]
```

Comment:
It took me a bit to realize there's an implied for loop variable, `value` (or `bookmark` etc).

That does seem reasonable, though I think `for each selected tab` should still assign `tab`.

But I think it would be good to also allow variable names. Specifically something like `tab` could easily get eaten up by the parser. E.g., `find tab` may not parse with "tab" in a slot (it actually might depending on the phrase matcher... but it's hard to be sure).

I'd suggest the phrasing `for each value in clipboard (as | named | as name) [variable] [dataSource=clipboard]`. This would require duplicating each of these phrases here.